core/build.gradle

```diff
@@ -137,6 +137,11 @@ dependencies {
     // there is a compatibility problem between the Guava Library used by HDFS, Google Cloud Storage and Reflections; the only version that fits for all is the following
     compile group: 'com.google.guava', name: 'guava', version: '20.0'
 
+    compile group: 'org.apache.arrow', name: 'arrow-vector', version: '4.0.0'
+    compile group: 'org.apache.arrow', name: 'arrow-memory-netty', version: '4.0.0'
```

Comment:
@JMHReif @fbiville I thought about this and maybe these two makes sense change to `compileOnly` as it is into core WDYT?

Reply:
`compileOnly` sounds like a good idea 👍